cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -837,5 +839,308 @@ define([
+            expect(this.view.$(SELECTORS.usageUnit)).not.toExist();
+            expect(this.view.$(SELECTORS.usageCount))
+                .toContainText('Not in Use');
+        });
+
+        it('should show non-empty usage appropriately', function() {
+            var usageUnitAnchors;
+
+            this.model.set('usage', [
+                {'label': 'label1', 'url': 'url1'},
+                {'label': 'label2', 'url': 'url2'}
+            ]);
+
+            this.model.set('showContentGroupUsages', false);
+            this.view.$('.show-groups').click();
+
+            usageUnitAnchors = this.view.$(SELECTORS.usageUnitAnchor);
+
+            expect(this.view.$('li.action-delete')).toHaveAttr(
+                'data-tooltip', 'Cannot delete when in use by a unit'
+            );
+            expect(this.view.$('.delete')).toHaveClass('is-disabled');
+            expect(this.view.$(SELECTORS.usageCount)).not.toExist();
+            expect(this.view.$(SELECTORS.usageText))
+                .toContainText('This content group is used in:');
+            expect(this.view.$(SELECTORS.usageUnit).length).toBe(2);
+            expect(usageUnitAnchors.length).toBe(2);
+            expect(usageUnitAnchors.eq(0)).toContainText('label1');
+            expect(usageUnitAnchors.eq(0).attr('href')).toBe('url1');
+            expect(usageUnitAnchors.eq(1)).toContainText('label2');
+            expect(usageUnitAnchors.eq(1).attr('href')).toBe('url2');
+        });
+
+        it('should hide non-empty usage appropriately', function() {
+            this.model.set('usage', [
+                {'label': 'label1', 'url': 'url1'},
+                {'label': 'label2', 'url': 'url2'}
+            ]);
+            this.model.set('showContentGroupUsages', true);
+            this.view.$('.hide-groups').click();
+
+            expect(this.view.$('li.action-delete')).toHaveAttr(
+                'data-tooltip', 'Cannot delete when in use by a unit'
+            );
+            expect(this.view.$('.delete')).toHaveClass('is-disabled');
+            expect(this.view.$(SELECTORS.usageText)).not.toExist();
+            expect(this.view.$(SELECTORS.usageUnit)).not.toExist();
+            expect(this.view.$(SELECTORS.usageCount))
+                .toContainText('Used in 2 units');
+        });
+
+        it('should hide validation icons and messages appropriately', function() {
+            this.model.set('usage', [
+                {'label': 'label1', 'url': 'url1'},
+                {'label': 'label2', 'url': 'url2'}
+            ]);
+            this.model.set('showContentGroupUsages', true);
+            this.view.$('.hide-groups').click();
+
+            expect(this.view.$(SELECTORS.usageUnitMessage)).not.toExist();
+            expect(this.view.$(SELECTORS.usageUnitWarningIcon)).not.toExist();
+            expect(this.view.$(SELECTORS.usageUnitErrorIcon)).not.toExist();
+        });
+    });
+
+    describe('Content groups editor view', function() {
```

Comment:
This whole section also appears to be a duplicate of 'Experiment group configurations editor view'. Please share as much logic as possible.